GUI.py

```diff
@@ -10,23 +10,66 @@
 from .fit import *
 import configparser
 
-language_path = os.path.realpath(__file__)[:-6] + "language\\"

```

Comment:
Using backslashes is only for Windows, I changed this to be platform independent